client/my-sites/upgrades/domain-management/transfer/index.jsx

```diff
@@ -17,6 +17,15 @@ class Transfer extends React.Component {
 	render() {
 		const slug = this.props.selectedSite.slug,
 			domainName = this.props.selectedDomainName;
+
+		let transferToAnotherUser = null;
```

Comment:
Let's avoid the let here. I would recommend inlineing, or making a helper method:
```jsx
const canTransferDomain = ! isAutomatedTransfer || ! selectedSite;
//...
{ canTransferDomain && <VerticalNavItem path={ paths.domainManagementTransferToAnotherUser( slug, domainName ) }> //...
```

or
```jsx
renderTransferDomain() {
    const { selectedSite } = this.props;
    //...
    if ( isAutomatedTransfer ||  ! selectedSite ) {
         return null;
    }
    return <VerticalNavItem //...
}
//... and in render
{ this.renderTransferDomain() }
```